docs/Reference/Server.md

```diff
@@ -73,15 +73,14 @@ describes the properties available in that options object.
     - [serializerCompiler](#serializercompiler)
     - [schemaErrorFormatter](#schemaerrorformatter)
     - [schemaController](#schemacontroller)
-      - [Ajv 8 as default schema validator](#ajv-8-as-default-schema-validator)
```

Comment:
why was this removed?

Reply:
It was an orphan link. [VSCode plugin for Markdown](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) I have on my machine removed it automatically.